packages/@glimmer/application/src/environment.ts

```diff
@@ -26,6 +26,28 @@ export interface EnvironmentOptions {
   appendOperations?: DOMTreeConstruction;
 }
 
+export function iterableFor(ref: Reference<Opaque>, keyPath: string): OpaqueIterable {
```

Comment:
Seems like this should be in `iterable.ts` rather than `environment.ts`?

Reply:
Moved as well.